content/develop/use/patterns/indexes/index.md

```diff
@@ -158,6 +178,14 @@ When representing much larger numbers, you need a different form of indexing
 that is able to index numbers at any precision, called a lexicographical
 index.
 
+## Time series indexes
+
+When you create a new time series using the [`TS.CREATE`]({{< baseurl >}}/commands/ts.create) command, you can associate one or more `LABELS` with it. Each label is a name-value pair, where the value can either be text or numeric. Labels serve as a secondary index that you can query using various time series commands.
```

Comment:
Strictly speaking label value is just text (even if it is a numeric value - it is actually text). I would change to name-value pair where the name and the value are text.